frontend/src/static/css/style.scss

```diff
@@ -145,6 +130,10 @@ header {
       }
 
       &__failed-repo {
+        .fa-exclamation {
+          @include mini-font;
```

Comment:
will this be applied to the latest `font-awesome-icon(icon="exclamation")`?

Reply:
Yes it is being applied. Can check in the preview by changing `mini-font` to `large-font`.